app/console.go

```diff
@@ -35,30 +37,36 @@ func RunConsole(ps *peer.PeerStore) {
 	shell.AddCmd(&ishell.Cmd{
 		Name: "address",
 		Help: "show the address this host is listening on",
-		Func: listenAddr,
+		Func: func(ctx *ishell.Context) {
+			listenAddr(ctx, peerStore)
+		},
 	})
 	shell.AddCmd(&ishell.Cmd{
 		Name: "peers",
 		Help: "show the peers this host is connected to",
-		Func: peers,
+		Func: func(ctx *ishell.Context) {
```

Comment:
The original situation was making `app` and `peerStore` global, which seemed less than ideal to me.